### ISSUE TITLE
Admin and developer commands

### ISSUE THREAD

**Issue by lukruh** [XaverStiensmeier/ilarisdiscordbot#68]
Do we "need" another cog for admin commands? I think I read somewhere that checks can be applied to cog groups (directly or via load_ext()). I'd suggest to add an `admins` key with list of id's and/or the name of the `admin_role` to the bot settings and provide a list of commands to users who are listed there or got the role. 

> If we use roles, we have to make sure that only server wide commands can be executed and no data from other servers accessed.

Maybe 
- !backup
- !language
- !restart
- !debug or !log
- !download

I've seen logging commands in other applications, that can activate debug mode and send you all logs for a certain time or until disabled. Don't know if we need any of this..

**Comment by XaverStiensmeier** [comment]
my first question is: Is admin a defined term by discord that always means the bot admin and not the server admin? If not, I would go for a different less ambiguous name like "bot_admin" to avoid confusion.

I feel like all of this is not high priority but definitely "nice to have".

Maybe an issue on its own, but: we should consider a command like `!bug` that you can use in order to add the last X lines in the chat to a bug report or something like that. Might be easier for a lot of people then using github. In the easiest implementation, it just prints the github issue link though.

**Comment by lukruh** [comment]
good call. Created #70

**Comment by lukruh** [comment]
I agree, we should use different terms for different admins.. instead of server_admin and bot_admin I'd suggest admin and developer. But I'm happy with both.. as long as we keep using the same terms in the code.

**Comment by XaverStiensmeier** [comment]
admin and developer sounds great.

**Comment by lukruh** [comment]
We could create a new cog/group for this and use the group wide interaction and context check methods:
https://fallendeity.github.io/discord.py-masterclass/cogs/#cog_check-method
